.github/workflows/ruby.yml

```diff
@@ -4,21 +4,14 @@ on:
   push:
     branches:
       - master
+
+env:
+  BUNDLE_WITHOUT: development
+
 jobs:
   rubocop:
-    runs-on: ubuntu-latest
-    steps:
-      - run: sudo apt-get update
-      - run: sudo apt-get install build-essential libsystemd-dev
-      - uses: actions/checkout@v2
-      - name: Setup Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: 2.5
-          bundler-cache: true
-      - name: Run rubocop
-        if: github.event_name != 'push'
-        run: bundle exec rubocop
+    uses: ekohl/actions/.github/workflows/rubocop.yml@v0
```

Comment:
It is my intention to move this to `theforeman` and `v1` before this should be merged.